themes/custom/droopler_theme/scss/components/_d_product_page.scss

```diff
@@ -3,7 +3,6 @@
     .node__content {
       .header {
         .title {
-          margin-top: 0.8rem;
```

Comment:
@mariusz-andrzejewski-droptica chyba się źle zrozumieliśmy :) ty tu usuwasz margin nad tytułem.. a mi tam w kometnarzu do taska chodził o to, że ta strona produktu jest bardzo scisnieta z menu:
może by tu wcisnąc jakiś odstęp, bo teraz ten button "ask for product" jest sklejony z menu.. 
może by to tak ciut rezepchać ? :) 
![image](https://user-images.githubusercontent.com/37533838/81291409-fbe0c900-9069-11ea-8bfe-1510005caae1.png)

Reply:
vs to co teraz : 
![image](https://user-images.githubusercontent.com/37533838/81291485-203ca580-906a-11ea-894b-19f3af15730f.png)

Reply:
W sumie sie dobrze zrozumieliśmy, ja ten margin usunałem dodatkowo, bo był dosc brzydki offset vertyklany miedzy CTA a titlem, CTA wyglada jakby byl podciagniety do gory.
Odstep całego kontenera jest w htmlu - zmiana klasy.

Reply:
aaa, ok ok, nie zauważyłem, jak jest to gites